### ISSUE TITLE
Security vulnerability in Jackson dependency

### ISSUE THREAD

**Issue by kelvinqian00** [dakrone/cheshire#190]
Recently a new vulnerability, CVE-2020-36518, was reported in Jackson:
> jackson-databind before 2.13.0 allows a Java StackOverflow exception and denial of service via a large depth of nested objects.

The fix would be to update Cheshire's Jackson dependency to the latest version, 2.13.2. As an interim solution, we updated our transitive Jackson dependencies in our applications as so:
```clojure
{:deps
  cheshire/cheshire
  {:mvn/version "5.10.1"
   :exclusions
   [com.fasterxml.jackson.core/jackson-core
    com.fasterxml.jackson.dataformat/jackson-dataformat-smile
    com.fasterxml.jackson.dataformat/jackson-dataformat-cbor]}
  com.fasterxml.jackson.core/jackson-core  {:mvn/version "2.13.2"}
  com.fasterxml.jackson.dataformat/jackson-dataformat-smile
  {:mvn/version "2.13.2"}
  com.fasterxml.jackson.dataformat/jackson-dataformat-cbor
  {:mvn/version "2.13.2"}
...}
```
but it is an inelegant solution compared to being able to update Cheshire directly.

**Comment by kelvinqian00** [comment]
**UPDATE: ** The CVE was updated and thus re-appeared in our application's NVD scans. As such, we had to update the `jackson-databind` sub-dependency separately to 2.13.2.1:
```clojure
{:deps
  cheshire/cheshire
  {:mvn/version "5.10.1"
   :exclusions
   [com.fasterxml.jackson.core/jackson-core
    com.fasterxml.jackson.dataformat/jackson-dataformat-smile
    com.fasterxml.jackson.dataformat/jackson-dataformat-cbor
    com.fasterxml.jackson.core/jackson-databind]}
  com.fasterxml.jackson.core/jackson-core  {:mvn/version "2.13.2"}
  com.fasterxml.jackson.dataformat/jackson-dataformat-smile
  {:mvn/version "2.13.2"}
  com.fasterxml.jackson.dataformat/jackson-dataformat-cbor
  {:mvn/version "2.13.2"}
  com.fasterxml.jackson.core/jackson-databind
  {:mvn/version "2.13.2.1"}
...}
```

GitHub issue regarding the CVE: https://github.com/FasterXML/jackson-databind/issues/2816

**Comment by borkdude** [comment]
Ping @dakrone

**Comment by dakrone** [comment]
Thanks for the ping @borkdude, I've bumped the dependencies and released 5.11.0.

**Comment by borkdude** [comment]
@dakrone Thanks a bunch!
